docs/robots/husky/tutorials_husky.mdx

```diff
@@ -4,7 +4,1535 @@ sidebar_position: 4
+is a great way for most users to learn more about their Husky; understanding how to effectively operate Husky
+in simulation is valuable whether you are in the testing phase with software you intend to ultimately deploy on a
+physical Husky or you do not have one and are simply exploring the platform's capabilities. 
+[Driving Husky](#driving-husky) covers how to teleoperate Husky using the remote control, a well as safety
+procedures for operating the physical robot. Anyone working with a physical robot should be familiar with this section.
+
+[Navigating Husky](#navigating-husky) is a follow-on to what is learned in the [Simuation](#simulating-husky)
+tutorial, as navigation and map-making may be run in the simulated environment. However, this content is applicable
+to both the simulator and the real platform, if your Husky is equipped with a laser scanner.
+
+[Testing Husky](#testing-husky) outlines how to validate that your physical Husky is working correctly.
+
+[Advanced Topics](#advanced-topics) covers items that are only required in atypical situations.
+
+---
+
+## Husky ROS Packages   {#husky-ros-packages}
+
+Husky fully supports ROS; all of the packages are available in [Husky Github](https://github.com/husky).
+
+### Description Package
+
+The [husky_description](https://github.com/husky/husky/tree/noetic-devel/husky_description) repository
+provides a [URDF](http://wiki.ros.org/urdf) model of Husky.
+
+Husky's URDF model can be visualized in rviz. Once you have installed the desktop software in an upcoming tutorial,
+you will be able to run:
+
+```bash
+roslaunch husky_viz view_model.launch
+```
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/husky_images/husky_view_model.png"
+      width="600"
+    />
+    <figcaption>Husky model</figcaption>
+  </figure>
+</center>
+
+Husky can be customized and extended through the use of several environment variables. These are summarized in the
+[README](https://github.com/husky/husky/blob/noetic-devel/husky_description/README.md) file.
+
+### Key ROS Nodes and Topics
+
+#### Nodes
+
+You can use `rosnode list` to see all the nodes running by default on a Husky computer.
+The most important nodes are summarized in the following table.
+
+| Node                     | Description                                                                                                                                |
+| :----------------------- | :----------------------------------------------------------------------------------------------------------------------------------------- |
+| `/husky_node`            | Provides control and communication between the Husky platform and ROS. Accepts velocity commands and provides system feedback on `/status` |
+| `/robot_state_publisher` | Subscribes to `/joint_states` and publishes the robot's state to `tf`                                                                      |
```

Comment:
Should this be /tf ?
I am not sure what is correct.
The way you worte it matches the existing Husky LaTex manual.

Reply:
Made it /tf